lib/events.js

```diff
@@ -257,8 +257,9 @@ function _addListener(target, type, listener, prepend) {
       if (m && m > 0 && existing.length > m) {
         existing.warned = true;
         const w = new Error('Possible EventEmitter memory leak detected. ' +
-                            `${existing.length} ${type} listeners added. ` +
-                            'Use emitter.setMaxListeners() to increase limit');
+                            `${existing.length} ${type.toString()} listeners ` +
```

Comment:
@lpinca No, currently values like `null` are allowed as types. I've actually used numbers before for the event name as part of a generic protocol parser.